Sources/Jaeger/Jaeger/Models/JaegerSpan.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  JaegerSpan.swift
+//  Jaeger
+//
+//  Created by Simon-Pierre Roy on 10/29/18.
+//
+
+import Foundation
+
+/**
+ The Jaeger version of an OpenTracing Span.
+ 
+ See the [Jaeger.Thrift](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/jaeger.thrift) definition.
+ */
+public struct JaegerSpan {
+    
+    /**
+     Creates a Jaeger Span from an OpenTracing Span.
+     
+     - Parameter span: An OpenTracing Span.
+     */
+    init(span: Span) {
```

Comment:
We talked about including a performance test for this and a couple other constructors. What happened to that?

Reply:
I will do that!

Reply:
Done!